code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm

```diff
@@ -20,8 +20,12 @@
 	/// The amount of time it takes to create a venus human trap.
 	var/growth_time = 120 SECONDS
 
+	var/growth_timer
+	var/obj/effect/countdown/flower_bud/countdown
```

Comment:
These need to be documented as per our [documentation guidelines](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)